Tests/SwiftlyTests/InstallTests.swift

```diff
@@ -35,7 +35,7 @@ final class InstallTests: SwiftlyTests {
     /// Tests that `swiftly install a.b` installs the latest patch version of Swift a.b.
```

Comment:
Swift 5.6 isn't available on Ubuntu 22.04, so I updated these tests to use 5.7.x wherever 5.6 or older was used.